tests/test_profiling.py

```diff
@@ -39,3 +40,36 @@ def test_ascii_generator():
         'add_2', ['add_2'], ['add_2']]], ['add_2', ['add_2', ['add_2'], ['add_2']], ['add_2']]]]
     ascii_tree = generate_ascii_tree(sample)
     assert isinstance(ascii_tree, str)
+
+def test_track_leaks():
+    @track_leaks
+    def func_a():
+        return sum([1, 1])
+    
+    _, leaked = func_a()
+    assert leaked == 0
+    
+    # I have heard this is a memory leak, but I am not sure if it actually is
+    # this decorator sure thinks it is
+    # MAYBE create our own bad C mini extension with a memory leak and test it.
+    @track_leaks
+    def func_leak():
+        try:
+            raise Exception
+        except Exception as e:
+            etype, evalue, tb = sys.exc_info()
+            return tb
```

Comment:
Do you know why this is memory leak?
Do you think you could possibly describe why it is, to make sure that this leak will not be soon deprecated in newer versions of python
And then remove this comment / change it to description of what is happening

Reply:
Added a clearer description of what's going on there